metadata/add_plot_boundaries.Rmd

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
You are re-using variable names in this tutorial. I think we're trying to avoid that

Reply:
Okay